--- a/Modules/Setup.dist.orig	2018-01-31 14:24:37.303816413 -0500
+++ b/Modules/Setup.dist	2018-01-31 14:23:58.024818325 -0500
@@ -402,3 +402,19 @@
   -DMODULE_NAME$(SQLITE_MODULE_NAME)
 
 greenlet greenlet.c -I$(prefix)/include -L$(exec_prefix)/lib
+
+numpy.core.multiarray \
+numpy/multiarray/alloc.c                numpy/multiarray/convert.c             numpy/multiarray/hashdescr.c         numpy/multiarray/numpyos.c  \
+numpy/multiarray/array_assign_array.c   numpy/multiarray/convert_datatype.c    numpy/multiarray/item_selection.c    numpy/multiarray/python_xerbla.c    \
+numpy/multiarray/array_assign.c         numpy/multiarray/ctors.c               numpy/multiarray/iterators.c         numpy/multiarray/refcount.c \
+numpy/multiarray/array_assign_scalar.c  numpy/multiarray/datetime_busday.c     numpy/multiarray/mapping.c           numpy/multiarray/scalarapi.c        \
+numpy/multiarray/arrayobject.c          numpy/multiarray/datetime_busdaycal.c  numpy/multiarray/methods.c           numpy/multiarray/sequence.c \
+numpy/multiarray/buffer.c               numpy/multiarray/datetime.c            numpy/multiarray/multiarraymodule.c  numpy/multiarray/shape.c    \
+numpy/multiarray/calculation.c          numpy/multiarray/datetime_strings.c    numpy/multiarray/nditer_api.c        numpy/multiarray/ucsnarrow.c        \
+numpy/multiarray/cblasfuncs.c           numpy/multiarray/descriptor.c          numpy/multiarray/nditer_constr.c     numpy/multiarray/usertypes.c        \
+numpy/multiarray/common.c               numpy/multiarray/dtype_transfer.c      numpy/multiarray/nditer_pywrap.c     numpy/multiarray/vdot.c     \
+numpy/multiarray/compiled_base.c        numpy/multiarray/flagsobject.c         numpy/multiarray/number.c        \
+numpy/multiarray/conversion_utils.c     numpy/multiarray/getset.c              numpy/multiarray/numpymemoryview.c \
+-Inumpy/include \
+-Inumpy/src \
+-I$(prefix)/include \
